spec/lib/efile/nj/nj1040_calculator_spec.rb

```diff
@@ -265,4 +265,126 @@ def over_65_birth_year
       expect(instance.lines[:NJ1040_LINE_42].value).to eq(instance.lines[:NJ1040_LINE_39].value)
     end
   end
+  
+  describe 'line 65 - NJ child tax credit' do
+    context 'when taxpayer is married filing separately' do
+      let(:intake) { create(:state_file_nj_intake, :married_filing_separately) }
+      it 'returns nil' do
+        expect(instance.lines[:NJ1040_LINE_65].value).to eq(nil)
+      end
+    end
+
+    context 'when taxable income is over 80k' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_one_dep) }
+      it 'returns nil' do
+        allow(instance).to receive(:number_of_dependents_age_5_younger).and_return 10
+        allow(instance).to receive(:calculate_line_42).and_return 80_001
+        instance.calculate
+        expect(instance.lines[:NJ1040_LINE_65].value).to eq(nil)
+      end
+    end
+
+    context 'when taxable income is under 80k and taxpayer has 1 dependent' do
+      let(:intake) { create(:state_file_nj_intake, :df_data_one_dep) }
+      it 'returns 1000 for incomes less than or equal to 30k' do
+        allow(instance).to receive(:number_of_dependents_age_5_younger).and_return 1
```

Comment:
[pebble] if you're going to do this for multiple blocks i would remove these lines from the it blocks and put them in a before block in the wrap around context so after line 288 add:
```
before do
allow(instance).to receive(:number_of_dependents_age_5_younger).and_return 1
end
```